7_react/react1/week3/hw-react-week3/src/components/Checkbox.jsx

```diff
@@ -0,0 +1,24 @@
+import "../App.css";
+
+const Checkbox = (props) => {
+  const {
```

Comment:
You could also directly "unpack" the props object in the line above to reduce the number of code lines, but of course this is also totally correct. 

See here: https://www.tutorialspoint.com/destructuring-of-props-in-reactjs